.changeset/pink-countries-camp.md

```diff
@@ -0,0 +1,20 @@
+---
+"perseus-build-settings": minor
+"@khanacademy/perseus-dev-ui": minor
+"@khanacademy/kas": minor
+"@khanacademy/keypad-context": minor
+"@khanacademy/kmath": minor
+"@khanacademy/math-input": minor
+"@khanacademy/perseus": minor
+"@khanacademy/perseus-core": minor
+"@khanacademy/perseus-editor": minor
+"@khanacademy/perseus-linter": minor
+"@khanacademy/perseus-score": minor
+"@khanacademy/perseus-utils": minor
+"@khanacademy/pure-markdown": minor
+"@khanacademy/simple-markdown": minor
+"jsdiff": minor
+"raphael": minor
```

Comment:
Hmm I suspect jsdiff and raphael are not necessary / useful for our testing purposes